consumer/consumer.go

```diff
@@ -22,6 +22,9 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector/internal/data"
 )
 
+// BaseMetricsConsumer defines a common interface for MetricsConsumer and MetricsConsumerV2
+type BaseMetricsConsumer interface{}
```

Comment:
This doesn't appear to be used anywhere.

Reply:
It will be used in createMetricsReceiver later on, but removed it for now